nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/factory/NDArrayFactory.java

```diff
@@ -1416,12 +1416,6 @@
 
     // =========== Sparse methods ===========
 
-    INDArray createSparseCSR(double[] data, int[] columns, int[] pointerB, int[] pointerE, long[] shape);
-
-    INDArray createSparseCSR(float[] data, int[] columns, int[] pointerB, int[] pointerE, long[] shape);
-
-    INDArray createSparseCSR(DataBuffer data, int[] columns, int[] pointerB, int[] pointerE, long[] shape);
-
     INDArray createSparseCOO(double[] values, long[][] indices, long[] shape);
```

Comment:
why this method isn't deleted together with others?

Reply:
It will be deleted as I continue refactoring.